### DISCUSSION TITLE
How to update account bio via the API?

### DISCUSSION THREAD

**Discussion by ShredderWalnut** [mastodon/mastodon#17296] [Q&A] [answered]
Hello,

I thought it would be neat to update the account bio programmatically via the API, so that I can display in my bio some data points retrieved from another API. For example, periodically run a script that retrieves a list of books I’m currently reading, and choose one at random to post on my Mastodon bio.

I skimmed through [the documentation](https://docs.joinmastodon.org), and I didn’t find a way to do this in the docs. Before opening a GitHub issue for a feature request, I figured I should ask here to be sure. Thank you in advance!

**Comment by trwnh** [comment, marked as answer]
`PATCH /api/v1/accounts/update_credentials` accepts the `note` parameter which represents the account bio: https://docs.joinmastodon.org/methods/accounts/

**Reply by ShredderWalnut** [reply]
Thank you so much, @trwnh!
